.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!---
+Verify first that your issue/request is not already reported on GitHub.
+-->
+
+##### ISSUE TYPE
+
+This issue tracker is only for the bug reports, for different issues see:
+
+ - Feature Idea: Please report feature ideas in the
+   [RFC repo](https://github.com/LibrePCB/librepcb-rfcs)
+ - Documentation Report: Please report documentation issues in the
+   [documentation repo](https://github.com/LibrePCB/librepcb-doc)
+ - Bug Report: You have come to the right place!
```

Comment:
I'm not sure if I should place this whole section in a HTML comment, so it won't show up in the actual bug report. On the other hand this makes it more visible.